cumulusci/robotframework/tests/salesforce/populate.robot

```diff
@@ -0,0 +1,89 @@
+*** Settings ***
+Documentation
+...  This suite tests the `Populate Field` keyword.
+...  Specifically, it verifies that the keyword properly
+...  overwrites previous values. We had some bugs where
+...  new values were getting appended rather than replacing
+...  existing values.
+...
+...  This suite uses an opportunity, since the edit dialog
+...  has several different field types on it.
+
+Library   String
+Library   DateTime
+Resource  cumulusci/robotframework/Salesforce.robot
+
+Suite Setup     run keywords
+...  Open test browser
+...  Create opportunity
+Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+Create opportunity
+    [Documentation]
+    ...  Creates a test opportunity with a random name and a close
+    ...  date 30 days in the future. A reference to the created
+    ...  opportunity will be stored in the suite variable ${opportunity},
+    ...  and be return as the result of this keyword.
+
+    ${30 days from now}=  Get Current Date  increment=30 days  result_format=%Y-%m-%d
```

Comment:
Looks like you aren't actually using this.

Reply:
Hah! Good catch. That's a bug. I added the variable but forgot to use it in the call to `salesforce insert`.

Reply:
I've pushed a fix.